fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CatalogAnalyzer.java

```diff
@@ -80,5 +81,25 @@ public Void visitDropCatalogStatement(DropCatalogStmt statement, ConnectContext
 
             return null;
         }
+
+        @Override
+        public Void visitUseCatalogStatement(UseCatalogStmt statement, ConnectContext context) {
```

Comment:
parse and set the catalog name to UseCatalogStmt in here and then you can use catalogName in StmtExecutor.

Reply:
done